apps/core/components/ItineraryCard/LocationName.js

```diff
@@ -14,7 +14,7 @@ type Props = {|
 function LocationName({ data, style }: Props) {
   return (
     <Text style={style} numberOfLines={1}>
-      {data?.name}
+      {data?.name ?? ''}
```

Comment:
Sounds like we should accept `?React.Node` here, so we don't have to do this everywhere 🙈